runtime/onert/core/src/exec/Executors.cc

```diff
@@ -283,10 +283,9 @@ void Executors::execute(const IODescription &desc)
     throw std::runtime_error{"Cannot find edge for model input"};
   };
 
-  // Assumption: nnpkg inputs/outputs are not duplicated.
-  // TODO Unify _type_aware_quant_tensors, pkg_input_edge_tensors and pkg_output_edge_tensors into
-  // one
-  std::unordered_map<ir::IODesc, std::shared_ptr<EdgeTensor>> pkg_io_edge_tensors;
+  // Tensors of nnpkg inputs/outputs for type-aware quantization
+  std::unordered_map<ir::IODesc, std::shared_ptr<EdgeTensor>> pkg_input_quant_tensors;
+  std::unordered_map<ir::IODesc, std::shared_ptr<EdgeTensor>> pkg_output_quant_tensors;
```

Comment:
`pkg_io_edge_tensors` was separated into `pkg_input_quant_tensors` and `pkg_output_quant_tensors` in order to avoid duplication of `IODesc` of inputs and outputs. `IOIndex` of inputs and outputs can be the same if they are actually different tensors.